kuma/static/js/helpfulness.js

```diff
@@ -4,6 +4,10 @@
     var articleTracker = doc.location.pathname + '#answered-helpful';
     // this feature requires localStorage
     if (win.mdn.features.localStorage) {
+        // clear out any drafts from localStorage, now that the document has been saved
```

Comment:
I saw that this code was adding helpfulness and thought that was weird because I never added it, but I  must have been working from an older version of master. Thanks for the heads up!